en/appendices/5-1-upgrade-guide.rst

```diff
@@ -40,6 +40,10 @@ Http
 - ``Client`` now emits ``HttpClient.beforeSend`` and ``HttpClient.afterSend``
   events when requests are sent. You can use these events to perform logging,
   caching or collect telemetry.
+- ``Http\Server::terminate()`` was added. This method triggers the
```

Comment:
Think we need to mention that this isn't automatically called and point to the other section you added.